code/game/objects/items/weapons/storage/garment.dm

```diff
@@ -0,0 +1,113 @@
+/obj/item/storage/bag/garment
+	name = "garment bag"
+	desc = "A bag for storing extra clothes and shoes."
+	icon_state = "garment_bag"
+	slot_flags = NONE
+	max_w_class = WEIGHT_CLASS_NORMAL
+	max_combined_w_class = 200
+	storage_slots = 21
+	resistance_flags = FLAMMABLE
+	can_hold = list(/obj/item/clothing)
```

Comment:
I have some balance concerns here, it would probably be a good idea to add a blacklist for EVA suits, heavy armors, and tech armors from R&D like the reactive/ablative armors. Atm this can hold most of these suits including the Syndicate Blood-red

Reply:
Just, what would be the best way to go about this? Forbid every `/obj/item/clothing/suit/space` subtype, specific armors such as riot and so on? Bit tedious but doable.

Honestly these armors should be larger than just normal but that's an issue for another day.